### PR TITLE
Revert "Control-br update"

### PR DESCRIPTION
**Pull request by gasaraki00**
Reverts OpenKore/openkore#237

### CODE REVIEW

File: tables/bRO/control-br/config.txt [hunk 1]
```diff
@@ -29,7 +29,6 @@ poseidonServer 127.0.0.1
 poseidonPort 24390
 
 bindIp
-forceMapIP
```
Comment (gasaraki00) [inline tables/bRO/control-br/config.txt]: -forceMapIP

**Comment by lututui** [comment]
Why?

File: tables/bRO/control-br/config.txt (hunk 1 above)
Comment (flacrasec) [inline tables/bRO/control-br/config.txt]: same question why this need to revert?

**Comment by Henrybk** [comment]
@gasaraki00 Would you please explain theses changes and why do you want to commit them?

**Comment by itsrachelfish** [comment]
It's been a week without an explanation, going to close this for now.

**Pull request closed without merging**